types/index.d.ts

```diff
@@ -34,14 +34,14 @@ export interface SwipeableOptions {
 // Component Specific Props
 export interface SwipeableProps {
   nodeName?: string
-  innerRef?: ({}: HTMLElement) => void
+  innerRef?: (element: HTMLElement) => void
```

Comment:
this should probably also be `(element: HTMLElement | null) => void` as it also just forwards the regular `ref` call: https://github.com/dogfessional/react-swipeable/blob/master/src/index.js#L227

Reply:
I added this too, but I have no idea how to write a test that uses this function 🤷‍♂